### ISSUE TITLE
No module name 'pydbm' after pip install

### ISSUE THREAD

**Issue by Whisht** [chimera0/accel-brain-code#4]
I have installed `pydbm` using `pip`, and the Dependencies `numpy` as well as `cython`  was installed.
![image](https://user-images.githubusercontent.com/16497652/58531196-141e5600-8214-11e9-9691-b1dabc94841d.png)
But there is ` no module named pydbm` error.
![image](https://user-images.githubusercontent.com/16497652/58531244-3f08aa00-8214-11e9-8416-c6b8225de19f.png)
Here is `sys.path` 
![image](https://user-images.githubusercontent.com/16497652/58531516-2d73d200-8215-11e9-904a-a96c2d87f237.png)
It seems that `pydbm` miss sorce file in `python/site-packages`.
![image](https://user-images.githubusercontent.com/16497652/58531598-76c42180-8215-11e9-96cc-0c4ca793bfde.png)

**Comment by helloWorld20484096** [comment]
Is there anyone can solve this? I encounter the same problem too.

**Comment by chimera0** [comment]
I'am still investigating the reasons of how this happen.

If it's urgent, you can install this library with the following command:

```bash
cd Deep-Learning-by-means-of-Design-Pattern/
python setup.py bdist_wheel
pip install dist/pydbm-{version}-{hoge}.whl
```

Please try the above method after cloning this repository.

The essential problem solving takes a bit of a while.
I have a lot of catching up to do this week.

**Comment by chimera0** [comment]
I wrote a method for dealing with the trouble in [README](https://github.com/chimera0/accel-brain-code/tree/master/Deep-Learning-by-means-of-Design-Pattern#installation).

But I decided to leave the essential solution out of consideration from the point of view of cost performance. It is a less important issue and not so prompted widespread concern.

About 20 people who are forking may have noticed the problem of setuptools that we are currently observing, and eliminated the problem in the simple way described above, without asking a question.
